examples/heterogeneous-node-classification-with-hashgnn.ipynb

```diff
@@ -0,0 +1,534 @@
+    "\n",
+    "Now that we loaded and understood the data we want to analyze, we can move on to look at the tools for actually making the aforementioned genre predictions of the `UnclassifiedMovie` nodes.\n",
+    "\n",
+    "Since we want to predict discrete valued properties of nodes, we will use a [node classification pipeline](https://neo4j.com/docs/graph-data-science-client/current/pipelines/#_node_classification)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "48d11361",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an empty node classification pipeline\n",
+    "pipe, _ = gds.beta.pipeline.nodeClassification.create(\"genre-predictor-pipe\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "45cba958",
+   "metadata": {},
+   "source": [
+    "To be able to compare our accuracy score to the current state of the art methods on this dataset, we want to use the same test set size as in the Graph Transformer Network paper ([NIPS paper link](https://proceedings.neurips.cc/paper/2019/file/9d63484abb477c97640154d40595a3bb-Paper.pdf)).\n",
+    "We configure our pipeline accordingly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9aacdd72",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the test set size to be 79.6 % of the entire set of `Movie` nodes\n",
+    "_ = pipe.configureSplit(testFraction=0.796)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20449f74",
+   "metadata": {},
+   "source": [
+    "Please note that we would get a much better model by using a more standard train-test split, like 80/20 or so.\n",
+    "And typically this would be the way to go for real use cases."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e9d0dbcc",
+   "metadata": {},
+   "source": [
+    "### The HashGNN node embedding algorithm\n",
+    "\n",
+    "As the last part of the training pipeline, there will be an ML training algorithm.\n",
+    "If we use the `plot_keywords` directly as our feature input to the ML algorithm, we will not utilize any of the relationship data we have in our graph.\n",
+    "Since relationships would likely enrich our features with more valuable information, we will use a node embedding algorithm which takes relationships into account, and use its output as input to the ML training algorithm.\n",
+    "\n",
+    "In this case we will use the HashGNN node embedding algorithm which is new in in GDS 2.3.\n",
+    "Contrary to what the name suggests, HashGNN is not a supervised neural learning model.\n",
+    "It is in fact and unsupervised algorithm.\n",
+    "Its name comes from the fact that the algorithm design in inspired by that of graph neural networks, in that it does message passing interleaved with transformations on each node.\n",
+    "But instead of doing neural transformations like most GNNs, its transformations are done by locality sensitive min-hashing.\n",
```

Comment:
could add that it therefore does not require training because the hashfunctions are chosen independently of the data

Reply:
sure :)

Reply:
done